src/java.desktop/macosx/classes/com/apple/eio/FileManager.java

```diff
@@ -58,7 +58,7 @@ public class FileManager {
         loadOSXLibrary();
     }
 
-    @SuppressWarnings("removal")
+    @SuppressWarnings({"removal", "restricted"})
```

Comment:
There are several of these changes. One option might have been to just disable restricted warnings when building. But on a deeper look, I realized that in all these places we already disabled deprecation warnings for the use of security manager, so I also added a new suppression instead.

Reply:
Sounds reasonable.